lib/features/product/presentation/functions.dart

```diff
@@ -70,6 +70,7 @@ Future<void> _afterPurchaseModal(
   if (singleTicketPurchase) {
     await ticketsCubit.useTicket(
       product.id,
+      // The first eligible menu item is used as the default.
```

Comment:
We don't have this functionality as part of this PR, but I added a comment acknowledging this